extras/videoDrivers/SDL2/VideoSDL2.cpp

```diff
@@ -28,6 +29,21 @@
             PrintError(SDL_GetError()); \
     } while(false)
 
+namespace {
+template<typename T>
+struct SDLMemoryDeleter
+{
+    void operator()(T* p) const
+    {
+        if(p)
```

Comment:
check is not necessary, `SDL_free(nullptr)` is also valid like `free(nullptr)`

Reply:
Furthermore `unique_ptr` doesn't even call the deleter for a nullptr

Reply:
The former isn't documented, though. ´SDL_free()` may not use libc `free()`.
Didn't know about `unique_ptr` checking.